pymatgen/electronic_structure/cohp.py

```diff
@@ -1036,46 +1088,46 @@ def summed_orbital_icohp(self):
 
 
 class IcohpCollection(MSONable):
-    """Store IcohpValues.
+    """Collection of IcohpValues.
 
     Attributes:
         are_coops (bool): Boolean to indicate if these are ICOOPs.
         are_cobis (bool): Boolean to indicate if these are ICOOPs.
-        is_spin_polarized (bool): Boolean to indicate if the Lobster calculation was done spin polarized or not.
+        is_spin_polarized (bool): Boolean to indicate if the LOBSTER calculation was done spin polarized or not.
     """
 
     def __init__(
         self,
-        list_labels,
-        list_atom1,
-        list_atom2,
-        list_length,
-        list_translation,
-        list_num,
-        list_icohp,
-        is_spin_polarized,
-        list_orb_icohp=None,
-        are_coops=False,
-        are_cobis=False,
+        list_labels: list,
+        list_atom1: list[str],
+        list_atom2: list[str],
+        list_length: list[float],
+        list_translation: list[Vector3D],
+        list_num: list,  # TODO: more specific type
+        list_icohp: list[dict[Spin, IcohpValue]],
+        is_spin_polarized: bool,
+        list_orb_icohp: list[dict] | None = None,  # TODO: more specific type
+        are_coops: bool = False,
+        are_cobis: bool = False,
     ) -> None:
         """
         Args:
-            list_labels: list of labels for ICOHP/ICOOP values
-            list_atom1: list of str of atomnames e.g. "O1"
-            list_atom2: list of str of atomnames e.g. "O1"
-            list_length: list of lengths of corresponding bonds in Angstrom
-            list_translation: list of translation list, e.g. [0,0,0]
-            list_num: list of equivalent bonds, usually 1 starting from Lobster 3.0.0
-            list_icohp: list of dict={Spin.up: icohpvalue for spin.up, Spin.down: icohpvalue for spin.down}
-            is_spin_polarized: Boolean to indicate if the Lobster calculation was done spin polarized or not Boolean to
-                indicate if the Lobster calculation was done spin polarized or not
-            list_orb_icohp: list of dict={[str(Orbital1)-str(Orbital2)]: {"icohp":{Spin.up: icohpvalue for spin.up,
-                Spin.down: icohpvalue for spin.down}, "orbitals":[Orbital1, Orbital2]}}
-            are_coops: Boolean to indicate whether ICOOPs are stored
-            are_cobis: Boolean to indicate whether ICOBIs are stored.
+            list_labels: list of labels for ICOHP/ICOOP values.
+            list_atom1 (list[str]): atom names e.g. "O1".
+            list_atom2 (list[str]): atom names e.g. "O1".
+            list_length: list of lengths of corresponding bonds in Angstrom.
+            list_translation: list of cell translation vectors.
+            list_num: list of equivalent bonds, usually 1 starting from LOBSTER 3.0.0.
```

Comment:
This description is a bit unclear to me

Reply:
There were Lobster output formats where they had an additional field that is now removed. There, they indicated the number of equivalent bonds.

Reply:
Thanks, `the number of equivalent bonds` is much clearer to me :)

The original description `list of equivalent bonds` is a bit ambiguous, and I would update the docstring.